backend/app.ts

```diff
@@ -546,43 +563,43 @@ io.on(M.connectionMsg, function (socket: any): void {
 
   socket.on(
     M.checkHasOngoingGameMsg,
-    function ({ eloId }: { eloId: string }): void {
-      logReceivedMessage(M.checkHasOngoingGameMsg, { eloId });
-      if (!isValidEloId(eloId)) {
+    function ({ idToken }: { idToken: string }): void {
+      logReceivedMessage(M.checkHasOngoingGameMsg, { idToken });
+      if (!isValidIdToken(idToken)) {
         emitMessage(M.respondHasOngoingGameMsg, { res: false });
         return;
       }
-      const game = GM.getOngoingGameByEloId(eloId);
+      const game = GM.getOngoingGameByIdToken(idToken);
       emitMessage(M.respondHasOngoingGameMsg, { res: game !== null });
     }
   );
 
   socket.on(
     M.returnToOngoingGameMsg,
-    function ({ eloId }: { eloId: string }): void {
-      logReceivedMessage(M.returnToOngoingGameMsg, { eloId });
-      if (!isValidEloId(eloId)) {
+    function ({ idToken }: { idToken: string }): void {
+      logReceivedMessage(M.returnToOngoingGameMsg, { idToken });
+      if (!isValidIdToken(idToken)) {
         emitMessage(M.ongoingGameNotFoundMsg);
         return;
       }
-      clientEloId = eloId;
-      const game = GM.getOngoingGameByEloId(clientEloId);
+      clientIdToken = idToken;
+      const game = GM.getOngoingGameByIdToken(clientIdToken);
       if (!game) {
         emitMessage(M.ongoingGameNotFoundMsg);
         return;
       }
-      const idx = clientIndex(game, clientEloId);
+      const idx = clientIndex(game, clientIdToken);
       if (idx === null) {
         console.log("Client index not found");
         return;
       }
       game.arePlayersPresent[idx] = true;
       // When a client returns to a game, we need to update its socket id.
       game.socketIds[idx] = socket.id;
-      let gameWithoutEloIds = JSON.parse(JSON.stringify(game));
-      delete gameWithoutEloIds.eloIds;
+      let gameWithoutIdTokens = JSON.parse(JSON.stringify(game));
+      delete gameWithoutIdTokens.idTokens;
```

Comment:
This line used to delete the eloIds because elo IDs were private, and we didn't want to leak a player's elo ID to its opponent. Are idTokens still private in this implementation? Or is the private part stored by auth0?

Reply:
I would say they're about as private as elo IDs. For all intents and purposes, idTokens equate to user info access.